tests/tmp_data/sample_N100/running/0/model.out

```diff
@@ -0,0 +1,1955 @@
+                                                          Cloudy 17.02
```

Comment:
Should the *out files be part of the repo? Won't they change all the time and thus be part of many future commits?